File: challenges/2-idade-em-dias/python/ellenmariadev/idade_em_dias.py
idade = int(input('\nDigite uma idade: '))

ano = idade // 365
mes = (idade % 365) // 30
idade = (idade % 365) % 30

print('\n{} ano(s)'.format(ano))
print('{} mes(es)'.format(mes))
print('{} dia(s)'.format(idade))

File: challenges/2-idade-em-dias/javascript/edurevsky/idade-em-dias.js
import userInput from './input.js'

const YEAR_DAYS = 365
const MONTH_DAYS = 30

const daysInput = userInput()

const daysInt = parseInt(daysInput)

const years = daysInt / YEAR_DAYS

const months = (daysInt % YEAR_DAYS) / MONTH_DAYS

const days = (daysInt % YEAR_DAYS) % MONTH_DAYS

console.log(`${Math.round(years)} ano(s)`)
console.log(`${Math.round(months)} mes(es)`)
console.log(`${Math.round(days)} dia(s)`)
